tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -178,17 +171,6 @@ exports[`cloud_function_expects_inflight.w > stderr 1`] = `
 5 | | });
   | \\\\-^ Expected type to be \\"IFunctionHandler (inflight (str): void)\\", but got \\"preflight (str): str\\" instead
 
-
-error: Expected type to be \\"IQueueAddConsumerHandler (inflight (str): void)\\", but got \\"inflight (num): void\\" instead
```

Comment:
Regression?

Reply:
Fixed

Reply:
Err- I recovered the error so it's still produced, but the error makes less sense since it references `$Inflight1` now.